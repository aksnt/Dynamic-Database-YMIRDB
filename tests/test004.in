SET a 1 2 3
GET B
FORWARD B
FORWARD A
SET a 1 2 A
SET a a
LIST ENTRIES
