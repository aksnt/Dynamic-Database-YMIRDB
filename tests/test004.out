> ok

> no such key

> no such key

> no such key

> no such key

> not permitted

> a [1 2 3]
